backend/functions/lib/adapters/gateways/dxtr/offices.dxtr.gateway.ts

```diff
@@ -14,6 +14,11 @@ import { buildOfficeCode, getOfficeName } from '../../../use-cases/offices/offic
 
 const MODULE_NAME = 'OFFICES-GATEWAY';
 
+// Remove invalid divisions at the gateway rather than forcing the
+// more important use case code to include logic to remove them.
```

Comment:
For legacy codes, I would say this is more of a business logic issue than a persistence issue, but we can leave it for an axis of change in the future to decide whether/how to treat them differently than the invalid ones.